bin/ingest

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
```

Comment:
This is seemingly inconsistent with Ingester PSC, DK, OC post-refactor, but I'm not sure what the best name is. I *think* it should be `ingest-bulk`, but I'm not sure.